packages/client/src/views/PrintCertificate/VerifyCollector.tsx

```diff
@@ -123,6 +123,22 @@ class VerifyCollectorComponent extends React.Component<IFullProps> {
     }
   }
 
+  getCollectorType = () => {
+    const informantType = (
+      this.props.declaration.data.informant.informantType as string
+    ).toLowerCase()
+    const { collector } = this.props.match.params
+    // to extract the id of mother, father, bride, groom
+    if (
+      collector === 'informant' &&
+      ['bride', 'groom', 'mother', 'father'].includes(informantType)
+    ) {
+      return informantType
+    } else {
+      return collector
+    }
+  }
+
```

Comment:
The draftToGqlTransformer func produces the object named eventRegistrationInput whose 'informant' property doesn't seem to return anything except fhirId & fhirIdPatient for 'bride', 'groom', 'mother', 'father' these types of informant. 

We used the function because we wanted to bring the id and idType information which was available in the gql transformer.